tests/test_node.py

```diff
@@ -92,6 +124,36 @@ def test_ova_node_AN(data, id, text, type, date):
     assert node.userdata == {}
 
 
+sadface_data_SchemeNode = [
+    (
+        """
+        {
+            "id": "70447169-9264-41dc-b8e9-50523f8368c1",
+            "metadata": {},
+            "name": "Support",
```

Comment:
I think the scheme types [are lowercased in SADFace](https://github.com/Open-Argumentation/SADFace/blob/c17fd89e3f6355cbe9bc4bd320bb1461894e2752/examples/hangback/data.json#L53)